phpunit/imap/MailCollectorTest.php

```diff
@@ -43,6 +43,7 @@
 use NotificationTarget;
 use NotificationTargetSoftwareLicense;
 use NotificationTargetTicket;
+use Sabre\VObject\Property\Boolean;
```

Comment:
Please drop this line (dunno why, I'm not able to propose suggestions)

Reply:
It's done